<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="ActionConnect"/>
    <addaction name="ActionDisconnect"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="menuRefereceData">
    <property name="title">
     <string>&amp;Reference Data</string>
    </property>
    <addaction name="CurrenciesAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRefereceData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionConnect"/>
   <addaction name="ActionDisconnect"/>
   <addaction name="CurrenciesAction"/>
  </widget>
  <action name="ExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="CurrenciesAction">
   <property name="icon">
    <iconset>
     <normaloff>ic_fluent_currency_dollar_euro_20_filled.svg</normaloff>ic_fluent_currency_dollar_euro_20_filled.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Currencies</string>
   </property>
  </action>
  <action name="ActionConnect">
   <property name="icon">
    <iconset>
     <normaloff>ic_fluent_plug_disconnected_20_filled.svg</normaloff>ic_fluent_plug_disconnected_20_filled.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="ActionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>ic_fluent_plug_connected_20_filled.svg</normaloff>ic_fluent_plug_connected_20_filled.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
